src/reducers/index.js

```diff
@@ -86,7 +86,7 @@ export default (state = initialState, action) =>
         draft.settings = { ...draft.settings, ...action.payload };
         break;
       case actionTypes.ADD_TO_BAR:
-        draft.bar = new Set([...draft.bar, action.payload]);
+        draft.bar = [...new Set([...draft.bar, action.payload])];
```

Comment:
Using a set to de-dupe array elements is fine, but this thing still needs to be an array.